indexing-service/src/main/java/io/druid/indexing/worker/executor/ExecutorLifecycle.java

```diff
@@ -91,30 +111,6 @@ public void start()
       throw Throwables.propagate(e);
     }
 
-    // Spawn monitor thread to keep a watch on parent's stdin
```

Comment:
Like, restarting the actual OS? I think it's best to have some other way to do it, since I think people expect apps to be able to be lifecycled independently of the host OS